### ISSUE TITLE
Add dark mode

### ISSUE THREAD

**Issue by Petrosdevri** [Ananya2001-an/githubwars#21]
**Is your feature request related to a problem? Please describe.**
It would be interesting to add dark mode to the web app because many users prefer dark mode as their default (including me).

**Describe the solution you'd like** 
An automatic mode theme using CSS rules. Dark shades can be implemented using hex colours (you can go to https://www.w3schools.com/colors/colors_picker.asp and type 212121 to find out a full color list that can be utilised for the project). 

**Describe alternatives you've considered**
An alternative would be to create a button where user can change mode as per choice.

**Comment by Ananya2001-an** [comment]
That sounds great @Petrosdevri.  You can work on this.

**Comment by Petrosdevri** [comment]
I have completed the automatic dark mode CSS rule, do you think I need to add a navbar to toggle between themes or is that adequate?

**Comment by Ananya2001-an** [comment]
Yeah you can maybe add the dark mode toggle button at the footer.

**Comment by Petrosdevri** [comment]
Okay great, I will have finished it by tomorrow as I got an exam today and my time is limited due to studies.

**Comment by Ananya2001-an** [comment]
Sure please take your time no need to stress about it.
